services/src/main/java/io/scalecube/services/ServiceCall.java

```diff
@@ -288,7 +252,7 @@ public ServiceCall create() {
               Optional<Object> check =
                   toStringOrEqualsOrHashCode(method.getName(), serviceInterface, params);
               if (check.isPresent()) {
-                return check.get(); // toString, hashCode was invoked.
+                return check.get(); // toString, hashCode was been invoked.
```

Comment:
:| There's no such thing as `was been`,  u meant `has been` ?

Reply:
has being

Reply:
damn it i read it "has been")